README.md

```diff
@@ -33,7 +37,22 @@ The mock method takes three arguments:
 
 The mock method executes insert, then update, and finally findOne and returns the result of findOne.
 
-Each argument is optional and we have some smart behavior to handle non-existant arguments
+Each argument is optional and we have some smart behavior to handle non-existant arguments:
 - `insert`, if not specified we'll grab the first part of the `update` operator
 - `update`, if not specified we'll skip the update, if a single argument is specified, we'll use the id returned by our insert operation as the first argument (the query)
-- `find`, if not specified we'll use the id returned by our insert operation
\ No newline at end of file
+- `find`, if not specified we'll use the id returned by our insert operation
+
+**The `mockMulti` method takes four arguments:**
+
+- `insert` an array of arguments to be passed to the mockCollection's insert method
```

Comment:
Actually `insert` is an array of documents to be inserted, and could should also support nested arrays.

For example, suppose the user wishes to log a successful insert operation:

```javascript
// Perform a multiple insert operations and log each successful insert
var log = function (error, result) {
  if (!error && result)
    console.log('insert succeeded');
  else
    console.error(error || 'result is empty');
};
collection.mockMulti([[{name: 'joe'}, log], [{$name: 'sam'}, log])
// we would expect two log statements:
// 'insert succeeded'
// MongoError...
```